src/framework/uicomponents/qml/Muse/UiComponents/TextInputField.qml

```diff
@@ -117,6 +117,8 @@ FocusScope {
                 root.ensureActiveFocus()
             }
         }
+
+        onTriggered: root.ensureActiveFocus()
```

Comment:
It might be good to place a small comment here to explain what this does, because that might not be obvious if you didn't just read the description of this PR